include/libserver/command/proto/LobbyMessageDefines.hpp

```diff
@@ -285,22 +285,23 @@ struct LobbyCommandCreateNicknameNotify
 };
 
 //! Serverbound create nickname ok command.
-struct LobbyCommandCreateNicknameOK
+struct LobbyCommandCreateNickname
```

Comment:
this is called OK in the original game, should we stray from the nomenclature?

Reply:
When you committed the command name in CommandProtocol, you did it without the OK, I made this change so it fit the command name you wrote
https://github.com/Story-Of-Alicia/alicia-server/blob/09cf68c38f31f6024f9d306f788ebc370596d272/src/libserver/command/CommandProtocol.cpp#L42
Personally I believe it should keep the OK, but i leave it up to you

Reply:
It should totally be OK. I must have forgot about it. Fix it for me please.